### ISSUE TITLE
Raw_response option

### ISSUE THREAD

**Issue by ylemarechal** [gacou54/pyorthanc#38]
This code is working : 

```python
if __name__ == '__main__':
    client = pyorthanc.Orthanc('orthanc', timeout=200, return_raw_response=False)
    series = pyorthanc.find_series(client, {'Modality': 'SR'})
```

But this one is KO : 
```python
if __name__ == '__main__':
    client = pyorthanc.Orthanc('orthanc', timeout=200, return_raw_response=True)
    series = pyorthanc.find_series(client, {'Modality': 'SR'})
```
We get this error
```
TypeError: object of type 'Response' has no len()
```
from this [line](https://github.com/gacou54/pyorthanc/blob/8acb7898da4c23cfdb42b5c274ff8055cacdd173/pyorthanc/_find.py#L264C20-L264C39).

When raw_reponse option is enabled, methods using requests are ko when they are waiting for response.content and get response body instead. 

A possibility would be to keep raw_response in inner methods and return content (according to return_raw_response option) in higher levels. @gacou54, what do you think ?

**Comment by gacou54** [comment]
That's a very good point. Ideally, I would like the various utility functions to return the raw response, but this is unfortunately not possible in many cases. For example, `find_series()` accumulates results via several calls to Orthanc, which means that we would have several responses to collect (which is not very elegant).

For the moment, I suggest checking if it is a `return_raw_response`, and if it is `True`, we make a copy of the client with `return_raw_response=False`. What do you think @ylemarechal ?

**Comment by ylemarechal** [comment]
It is a good option.
Thank you

**Comment by gacou54** [comment]
Fixed in #42 

Bugfix has been released in PyOrthanc v1.12.3
